modules/dnsdist/README.md

```diff
@@ -118,23 +118,32 @@ jobs:
       X-API-Key: your-api-key # static pre-shared authentication key for access to the REST API (api-key).
 ```
 
+</details>
+
 ##### Multi-instance
 
-When you are defining more than one jobs, you must be careful to use different job names, to not override each other.
+> **Note**: When you define multiple jobs, their names must be unique.
```

Comment:
This note should go to Configuration->file (we can hardcode it in the template, no need to have it in meta)